src/slot.rs

```diff
@@ -855,29 +855,11 @@ impl<Conn: Connection> SetWorker<Conn> {
         loop {
             tokio::select! {
                 biased;
-                // If we should exit, terminate immediately
+                // If we should exit, terminate immediately.
```

Comment:
Whoops, I didn't need to add the period, my bad --- I can remove this if Git diff minimalism is important to you. :)

Reply:
nah, don't worry about it